File: Controllers/HomeController.cs
using System;
using Microsoft.AspNetCore.Mvc;

namespace newsgen_backend.Controllers
{
    /*
        <summary>
            API controller for verifying that the backend is working.
        </summary>
    */
    [ApiController]
    [Route("/")]
    public class WeatherForecastControllerCopy : ControllerBase {

        /*
            <summary>
                Verifies that the backend is working.
            </summary>
            <returns>
                Message.
            </returns>
        */
        [HttpGet]
        public String printMessage()
        {
            return "Backend API for Newsgen is working!";
        }
    }
}

File: src/main/java/com/newsgen/newsgenbackend/controller/HomeController.java
package com.newsgen.newsgenbackend.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * This API verifies that the application works.
 */
@RestController
public class HomeController {
    @GetMapping
    public String printMessage() {
        return "newsgen-backend API is working!";
    }
}
